main/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.urls import path
+from django.contrib.auth import views as auth_views
 from . import views
 
 urlpatterns = [
     path('', views.home, name='home'),
+    path('login/', auth_views.LoginView.as_view(template_name='main/login.html'), name='login'),
+    path('logout/', auth_views.LogoutView.as_view(template_name='main/logout.html'), name='logout'),
```

Comment:
Is this actually doing login and logout? Or is it just opening html pages for now saying that it is done?

Reply:
@simzacks It's logging in and out - fully functional.